server.go

```diff
@@ -1741,6 +1747,13 @@ func (s *Server) GetCurrentConcurrency() uint32 {
 	return atomic.LoadUint32(&s.concurrency)
 }
 
+// GetOpenConnectionsCount returns a number of opened connections.
+//
+// This function is intended be used by monitoring systems
+func (s *Server) GetOpenConnectionsCount() int32 {
+	return atomic.LoadInt32(&s.open) - 1
```

Comment:
Because of line: https://github.com/valyala/fasthttp/blob/3c7913fc29526508ce2e8724810cf1699826c1a4/server.go#L1542

So if I don't do `- 1` then I get a wrong value. I see `2` if there's only one incoming open connection.